src/messages.cs

```diff
@@ -203,7 +203,7 @@ interface MSMessageTemplateLayout
 
 	[iOS (10,0)]
 	[BaseType (typeof(NSObject))]
-	interface MSSession {}
+	interface MSSession : NSSecureCoding {}
```

Comment:
It's because the type is already part of the exception list, i.e. `case "MSSession":`